pkg/auth/migrate/rbac_to_acl.go

```diff
@@ -107,14 +109,55 @@ func RBACToACL(ctx context.Context, svc auth.Service, doUpdate bool, creationTim
 			if errors.Is(err, auth.ErrAlreadyExists) {
 				warnings = multierror.Append(warnings, err)
 			} else if err != nil {
-				return err
+				return nil, err
 			}
 		}
 
 		messageFunc(group.DisplayName, *newACL, warnings)
 	}
-
-	return nil
+	var usersWithPolicies []string
+	hasMoreUser := true
+	afterUser := ""
+	for hasMoreUser {
+		// get membership groups to user
+		users, userPaginator, err := svc.ListUsers(ctx, &model.PaginationParams{
+			After:  afterUser,
+			Amount: pageSize,
+		})
+		if err != nil {
+			return nil, err
+		}
+		for _, user := range users {
+			// get policies attracted to user
+			hasMoreGroupPolicy := true
+			afterGroupPolicy := ""
+			for hasMoreGroupPolicy {
+				userPolicies, userPoliciesPaginator, err := svc.ListUserPolicies(ctx, user.Username, &model.PaginationParams{
+					After:  afterGroupPolicy,
+					Amount: pageSize,
+				})
+				if err != nil {
+					return nil, fmt.Errorf("list user policies: %w", err)
+				}
+				if len(userPolicies) > 0 {
+					usersWithPolicies = append(usersWithPolicies, user.Username)
+				}
+				if !doUpdate {
+					break
+				}
+				for _, policy := range userPolicies {
+					if err := svc.DetachPolicyFromUser(ctx, policy.DisplayName, user.Username); err != nil {
+						return nil, fmt.Errorf("failed detaching policy %s from user %s: %w", policy.DisplayName, user.Username, err)
+					}
+				}
```

Comment:
```suggestion
				if doUpdate {
					for _, policy := range userPolicies {
						if err := svc.DetachPolicyFromUser(ctx, policy.DisplayName, user.Username); err != nil {
							return nil, fmt.Errorf("failed detaching policy %s from user %s: %w", policy.DisplayName, user.Username, err)
						}
					}
				}
```

Reply:
I don't want to go over all policies if in update mode